app/src/main/res/layout/fragment_tickets.xml

```diff
@@ -8,119 +8,125 @@
-                android:layout_width="0dp"
+                android:id="@+id/time"
+                android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_weight="1"
-                android:text="@string/price"
-                android:textColor="@color/black"
+                android:layout_margin="@dimen/layout_margin_medium"
                 android:textSize="@dimen/text_size_large"
-                tools:text="Price" />
+                tools:text="May 24 - May 25" />
+
+            <ProgressBar
+                android:id="@+id/progressBarTicket"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:elevation="@dimen/card_elevation"
+                android:visibility="gone" />
 
             <TextView
-                android:layout_width="0dp"
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_weight="1"
-                android:text="@string/qty"
+                android:id="@+id/ticketInfoTextView"
+                android:textAlignment="center"
+                android:visibility="gone"
+                android:text="No tickets available!"
+                android:padding="8dp"
+                android:layout_marginTop="8dp"
+                android:layout_marginLeft="@dimen/layout_margin_medium"
+                android:layout_marginRight="@dimen/layout_margin_medium"
+                android:background="@drawable/filled_border"
                 android:textColor="@color/black"
-                android:textSize="@dimen/text_size_large"
-                tools:text="Qty" />
-        </LinearLayout>
-        <androidx.recyclerview.widget.RecyclerView
-            android:id="@+id/ticketsRecycler"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center" />
+                android:textSize="@dimen/text_size_large"/>
 
-        <androidx.appcompat.widget.AppCompatButton
-            android:id="@+id/register"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center"
-            android:layout_margin="@dimen/layout_margin_large"
-            android:padding="@dimen/padding_medium"
-            android:backgroundTint="@color/colorAccent"
-            android:text="@string/register"
-            android:textColor="@android:color/white" />
-    </LinearLayout>
-</androidx.core.widget.NestedScrollView>
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:id="@+id/ticketTableHeader"
+                android:background="@drawable/filled_border"
+                android:layout_marginTop="8dp"
```

Comment:
In dimens.xml

Reply:
@nikit19 Will change once back to machine